controllers/v2/account_controller_v2.py

```diff
@@ -0,0 +1,40 @@
+
+
+
+from containers import Container
+from controllers.v2.abstract.abstract_create_read_controller_v2 import AbstractCreateReadControllerV2
+from controllers.v2.abstract.abstract_read_controller_v2 import AbstractReadControllerV2
+from dtos.v2.account_dto_v2 import AccountLoginRequestV2, AccountLoginResponseV2
+from services import account_service
+from services.account_service import AccountService
+from dependency_injector.wiring import inject, Provide
+
+
+
+class AccountsView(AbstractReadControllerV2):
```

Comment:
Login is an interesting case, we may not ever simply retrieve a user.  It might make more sense to inherit the MethodView class directly as opposed to the AbstractReadControllerV2

Reply:
As this AccountsView may need to incorporate other feature for Admin Account (like create / read), currently inherit from AbstractCreateReadController, we need to discuss more on the final features before deciding which class it inherit from

Reply:
Sounds good, worth further discussion!